pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py

```diff
@@ -689,17 +689,21 @@ def sort_legal_provision(elem):
         return sort_title, sort_number
 
     @staticmethod
-    def sort_hints_laws(elem):
+    def sort_by_index(elem):
         """
-        Provides the sort key for the supplied hint & law element as a tuple consisting of:
+        Provides the sort key for the supplied hint / law / legal provision
+        element as a tuple consisting of:
          * index
 
         Args:
-            elem (dict): one element of the hints list
+            elem (dict): one element of the hints / laws / legal provisions list
         Returns:
             sort key (tuple)
         """
-        return elem.get('index', 1000)
+        index = elem.get('Index', 1000)
```

Comment:
Why 1000, and not something really high, so that documents without index are sorted at the end for sure?

Reply:
Specs (https://models.geo.admin.ch/V_D/OeREB/OeREBKRM_V2_0.ili) define the index as range from -1000...1000.

Reply:
Ah OK, thanks.
Could you add that as a comment to the code? That way it will be completely clear why we use 1000.

Reply:
@jwkaltz : I guess that the confederation was not aware that 1000 would not suffice as they are lacking a pratical implementation...